packages/config-transformer/tests/Converter/ConfigFormatConverter/YamlToPhp/Fixture/normal/constant.yaml

```diff
@@ -23,4 +25,6 @@ return static function (ContainerConfigurator $containerConfigurator): void {
     $parameters->set('older_line_ending', PHP_EOL);
 
     $parameters->set('older_line_ending_with_space', PHP_EOL);
+
+    $parameters->set('class_constant', YamlToPhpTest::TEST);
```

Comment:
Could you extract a new test case fixture? Just in case this goes wrong in the future, so we have easier way to debug :)

Reply:
Done